pkg/pillar/cmd/volumemgr/updatestatus.go

```diff
@@ -446,8 +438,7 @@ func doUpdateVol(ctx *volumemgrContext, status *types.VolumeStatus) (bool, bool)
 	return changed, false
 }
 
-// updateStatus update all VolumeStatus/ContentTreeStatus which include a blob
-// that has this Sha256
+// Find all the VolumeStatus/ContentTreeStatus which refer to this Sha256
```

Comment:
This will fail a linter. The first word in the comment is supposed to be the name of the func that follows. Minor, but we should keep to it.

Reply:
Fixed